cms/static/cms/js/modules/cms.base.js

```diff
@@ -39,6 +39,8 @@ var CMS = {
     $(document).ready(function () {
         CMS.API.Helpers = {
 
+            isTouch: window.ontouchstart !== undefined,
```

Comment:
checking this every time is an overhead, I think. My thought is that if a user taps on the page at least once, it makes sense to suggest that he is going to keep tapping through the ui. So I've introduced the `isTouch` flag, which defaults to `false` and updates on first body tap